st2actions/st2actions/runners/actionchainrunner.py

```diff
@@ -464,6 +470,11 @@ def _render_publish_vars(action_node, action_parameters, execution_result,
         context.update(chain_vars)
         context.update({RESULTS_KEY: previous_execution_results})
         context.update({SYSTEM_SCOPE: KeyValueLookup(scope=SYSTEM_SCOPE)})
+        context.update({
+            DATASTORE_PARENT_SCOPE: {
```

Comment:
Yeah, I thought about this but there are cases where we want just SYSTEM_SCOPE inside DATASTORE_PARENT_SCOPE and in some cases we also want USER_SCOPE (config loader case). So then we need three get_system_datastore_jinja_context, get_user_datastore_jinja_context and get_datastore_jinja_context. It seemed more indirection to me.